### PR TITLE
Added Earlopain to maintainers list

### PR DESCRIPTION
**Pull request by nhtruong**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/backport.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   backport:
+    if: ${{ contains(github.event.label.name, 'backport') }}
```
Comment (VachaShah) [inline .github/workflows/backport.yml:10]: The workflow also needs updated parameters. You can find the correct implementation here: https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/backport.yml#L36

**Comment by bbarani** [comment]
Maintainer invitation sent to @Earlopain

**Comment by Earlopain** [comment]
Got it and accepted, thanks everyone (:

**Comment by Earlopain** [comment]
Got it and accepted, thanks everyone (:

**Review by dblock — APPROVED** [review]

**Pull request merged**